### Goal (arguments) definition

# Terrain to traverse
convex_plane_decomposition_msgs/PlanarTerrain terrain

# Start and goal configurations
SingleRigidBody start_state
SingleRigidBody goal_state

# Times to sample the trajectory for initial guesses
float64[] state_sample_times
---
### Result (return values) definition

# trajectory containing the generated trajectory at each timestep
fpowr/InitialGuessArray initial_guesses

# The contact data for each foot at each timestep
fpowr/FootstepPlan footstep_plan
---